base/strings/util.jl

```diff
@@ -224,6 +224,7 @@ end
     chopprefix(s::AbstractString, prefix::Union{AbstractString,Regex}) -> SubString
 
 Remove the prefix `prefix` from `s`. If `s` does not start with `prefix`, a string equal to `s` is returned.
+For case-insensitive comparison of the prefix, e.g. for shortening regular URLs, you can use a suitable `Regex` for `prefix`.
```

Comment:
```suggestion
Note that `prefix` can also be a regular expression, which makes this quite powerful. For example, one can use this to make the prefix comparison case-insensitive.
```

Reply:
The example comes below, so I don't see why one should point one it. Someone who needs more powerful matching will be happy about this comment. Someone who just reads the comment won't have to wonder about the importance of URL shortening in Julia ;-).

Whatever you do in the end, don't forget to adjust the `chopsuffix` docstring, too

Reply:
I'll probably you with comment, as is, maybe add, "as shown below"? I'll for sure drop "regular", maybe also URL shortening (from there) since implied.

Do you have any idea about the CI failure? A false alarm?